pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -20,15 +20,51 @@ import (
 	"fmt"
 	"strings"
 
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/validation"
 	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
+	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap"
 	"k8s.io/apiserver/pkg/util/shufflesharding"
 	apivalidation "k8s.io/kubernetes/pkg/apis/core/validation"
 	"k8s.io/kubernetes/pkg/apis/flowcontrol"
+	"k8s.io/kubernetes/pkg/apis/flowcontrol/install"
 )
 
+var mandatoryFlowSchemas = make(map[string]*flowcontrol.FlowSchemaSpec)
```

Comment:
I think the below pattern is very slightly safer:

```
var mandatoryFlowSchemas = func() map[string]*flowcontrol.FlowSchemaSpec {
  // all the work from the below
}
```

This way other init functions can't use it before it's set up :/

Reply:
done